openshift/templates/visual-verifier/visual-verifier-deploy.safe-entry-c19.param

```diff
@@ -5,6 +5,7 @@
 #=========================================================
 NAME=visual-verifier
 APP_NAME=visual-verifier
+NAMESPACE_NAME=a99fd4
 APP_GROUP=safe-entry-c19
 SUFFIX=-safe-entry-c19
 ROLE=visual-verifier
```

Comment:
Seems to be missing the new parameter

Reply:
I assume you're talking about OIDC host? if so it's been added now